index.d.ts

```diff
@@ -0,0 +1,75 @@
+// Type definitions for react-responsive-context
+// Project: react-responsive-context
+// Definitions by: Filipe Medeiros filipesilvamedeiros@gmail.com
+
+import { Component, FC, ReactElement, ReactNode } from "react";
+
+type BreakpointNames = 'xs' | 'sm' | 'md' | 'lg' | 'xl'
+
+type InitialBreakpoints = BreakpointNames | '_initial'
+
+type MediaQueries = {
+  [mq in InitialBreakpoints]?: string
+}
+
+type MediaTypeMapping = boolean
+
+type IsCalculated = boolean
+
+type MediaTypeMap = {
+  [mt in InitialBreakpoints]?: MediaTypeMapping
+}
+
+type Orientation = 'portrait' | 'landscape' | null
+
+interface ResponsiveContext {
+  isCalculated: IsCalculated,
+  mediaType: InitialBreakpoints,
+  lessThan: MediaTypeMap,
+  greaterThan: MediaTypeMap,
+  is: MediaTypeMap,
+  orientation: Orientation,
+}
+
+interface IsMobile {
+  isCalculated: boolean,
+  // Maybe this shouldn't be fixed on this screen size?
+  // Could just use
+  // isMobile: MediaTypeMapping
+  isMobile: ResponsiveContext['lessThan']['md'],
```

Comment:
This can't use "md".
Wait for https://github.com/Farfetch/react-context-responsive/pull/28 it includes a new prop "mobileBreakpoint" in which the consumer specifies which breakpoint is considered mobile

Reply:
I can wait for that PR to include that new property in the context, but for this LOC in specific, we can just change it to `boolean`/`MediaTypeMapping`

Reply:
Can you check the new version to see if you agree with it? Thanks!

Reply:
Seems fine now!